components/post_markdown/post_markdown.tsx

```diff
@@ -59,11 +59,12 @@ export default class PostMarkdown extends React.PureComponent<Props> {
     };
 
     getOptions = memoize(
-        (options: TextFormattingOptions | undefined, disableGroupHighlight: boolean, mentionHighlight: boolean | undefined) => {
+        (disableGroupHighlight: boolean, options?: TextFormattingOptions, mentionHighlight?: boolean, editedAt?: number) => {
```

Comment:
disableGroupHighlight, being a boolean, doesn't change much the semantics whether it is optional or not, unless we want to force the developer to write true or false for that, which I don't think it is the case.

Therefore, to lower the footprint of this pr, I would just make disableGroupHighlight an optional argument.

Reply:
yep ... good point!